### DISCUSSION TITLE
Geowebcache stand alone

### DISCUSSION THREAD

**Discussion by Abelon38** [hajkmap/Hajk#1468] [Q&A]
Hi there! 

I wonder if Geowebcache stand alone works with Hajk? I have tried with WMS and WMTS från my Geowebcacache (Stand alone) without success. Getting bad request whatever I do.  From Geoserver and QGIS-server WMS works well but not the stand alone version of GWC. I would appreciate any idea!

Thanks 
Abel

**Comment by jacobwod** [comment]
Generally speaking, I see no reason why responses from GeoServer's built-in GWC would differ from stand-alone GWC. Having said that, our organisation uses the built-in GWC, so I do not know for sure.

In GeoServer (and probably in GWC) you must setup the resolutions for a gridset properly. You must match that resolution gridset with the one you supply to Hajk (in your map's JSON config, see the `resolutions` array). My best guess is that there's some mismatch here. What does the response from GWC say?

**Reply by Abelon38** [reply]
Thanks Jacob! 

It seams that Hajk doesn't have support for GWC Stand alone yet. Today I spended some hours with a good colleague from Sweco and the conclusion was that some development is needed if we want to use WMS and WMTS from GWC in Hajk. 

I will be happy to see that development :)

Have great day!
BR/Abel

**Reply by jacobwod** [reply]
That's correct @Abelon38, my assumption was wrong. BTW, as a coincidence [this issue](https://github.com/hajkmap/Hajk/issues/1469) showed up yesterday. 😉

**Reply by sweco-semara** [reply]
@Abelon38 After our analysis that some work needs to be done in Hajk admin to support a stand-alone GWC server, Nacka has initiated an effort to fix this at least for their use-case and give back to the Hajk community. See issue #1469 linked by Jacob and it's upcoming PR.

**Reply by Abelon38** [reply]
Thanks for the update Martin! 😉
